### ISSUE TITLE
please add node

### ISSUE THREAD

**Issue by Mert-Zengin** [AntoshVik/BTCBreaker#3]
please add good api or make a node for check balance from localhost

**Comment by AntoshVik** [comment]
@Mert-Zengin , I do not think that this function will appear, the program was originally planned as a thin client without a local node.
